zephyr/port/boot/esp_loader.c

```diff
@@ -15,14 +15,16 @@
 #include "soc/soc_memory_layout.h"
 #include "esp_log.h"
 
-#ifndef CONFIG_SOC_SERIES_ESP32C3
+#if !defined(CONFIG_SOC_SERIES_ESP32C3) && !defined(CONFIG_SOC_ESP32C6)
 #include "soc/dport_reg.h"
```

Comment:
Would it be to bad adding `#include "soc/dport_reg.h"` into proper xtensa's #ifdef instead of making this alone-include here?

Reply:
We agreed to update those ifdef conditions on separate PR's